jobbergate-cli/jobbergate_cli/subapps/job_submissions/app.py

```diff
@@ -44,6 +44,11 @@ def create(
         ...,
         help="The id of the job_script from which to create the job submission",
     ),
+    cluster_id: str = typer.Option(
+        None,
+        "--cluster-name",
+        help="The name of the cluster where the job should be submitted (i.g. 'nash-staging')",
+    ),
```

Comment:
Why the name of the cluster is stored in a `cluster_id` variable instead of `cluster_name`?

Reply:
It is because the api expects a parameter named "cluster_id", but discussions around this pointed to using `--cluster-name` for the CLI to make it clearer for humans.